pext/ui_qt5.py

```diff
@@ -1557,7 +1557,7 @@ def __init__(self, main_loop_queue: Queue, window: 'Window') -> None:
             listener.start()
 
     def on_press(self, key) -> bool:
-        """Executed when a key is pressed."""
+        """Execute when a key is pressed."""
```

Comment:
This seems like bad grammar now. The old description was quite unclear anyway. Maybe something like this:

```suggestion
        """Track pressed keys and show window when all keys for the global hotkey are pressed."""
```

Reply:
Fixed in the commit https://github.com/airvzxf/Pext/commit/f765d42edac30a6f25f3e53c0791c76b7c61e31f.